test/mock-connector.py

```diff
@@ -59,27 +81,34 @@ def msg_consume(args):
     channel.start_consuming()
 
 def msg_update(args):
+    msg = __parse_update_message(args.json_msg_file)
     channel.basic_publish(exchange=fog_exchange,
-                          routing_key=KEY_UPDATE, body=args.message)
+                          routing_key=KEY_UPDATE, body=msg)
 def msg_request(args):
+    msg = __parse_request_message(args.json_msg_file)
     channel.basic_publish(exchange=fog_exchange,
-                          routing_key=KEY_REQUEST, body=args.message)
+                          routing_key=KEY_REQUEST, body=msg)
 
 parser = argparse.ArgumentParser(description='Mock KNoT Fog Connector')
 parser.set_defaults(func=msg_consume)
 subparsers = parser.add_subparsers(help='sub-command help', dest='subcommand')
 
 parser_update = subparsers.add_parser('send-update', help='Sends a message to \
     update the sensor in device')
-parser_update.add_argument('message', type=str, help='Update message to be \
-    sent. Format: {"id": <device_id>, "data":[{"sensor_id": <sensor_id>, \
-    "data": <sensor_data>}, ...]}')
+parser_update.add_argument('-f', '--json-msg-file', type=str, help='JSON File \
+    with update message to be sent. Format: {"id": <device_id>, \
+    "data":[{"sensor_id": <sensor_id>, \
+    "data": <sensor_data>}, ...]}',
+    default='', metavar="MSG_UPDATE_FILE")
 parser_update.set_defaults(func=msg_update)
 
 parser_request = subparsers.add_parser('send-request', help='Sends a message \
     requesting data from sensor device')
-parser_request.add_argument('message', type=str, help='Request message to be \
-    sent. Format: {"id": <device_id>, "data":[<sensor_id>, ...]}')
+parser_request.add_argument('-f', '--json-msg-file', type=str, help='JSON File \
+    with request message to be sent. Format: {"id": <device_id>, \
+    "data":[{"sensor_id": <sensor_id>, \
+    "data": <sensor_data>}, ...]}',
+    default='', metavar="MSG_REQUEST_FILE")
```

Comment:
`default=json.dumps(defaultMsgRequest), ...`

Reply:
If the default argument is not ' ', I do not have a manner to know if the user passes a file or not.